src/deepymod/model/library.py

```diff
@@ -55,31 +56,35 @@ def library_deriv(data: torch.Tensor, prediction: torch.Tensor, max_order: int)
 
 # ========================= Actual library functions ========================
 class Library1D(Library):
-    """ Construct a 1-dimensional library.
-
-    Args:
-        Library ([type]): Library object that can compute the library
-    """
     def __init__(self, poly_order: int, diff_order: int) -> None:
-        """ Create a library up with polynomial order containing up to 
-        differentials order. i.e. for poly_order=1, diff_order=3       
-        [$1, u_x, u_{xx}, u_{xxx}, u, u u_{x}, u u_{xx}, u u_{xxx}$]
+        """ Calculates the temporal derivative a library/feature matrix consisting of
+        1) polynomials up to order poly_order, i.e. u, u^2...
+        2) derivatives up to order diff_order, i.e. u_x, u_xx
+        3) cross terms of 1) and 2), i.e. Suu_xS, Su^2u_xxS
```

Comment:
small typo here, 'S' should be $ i guess?

Reply:
Indeed, fixed it!